spec/views/hyrax/raster_works/show.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "hyrax/raster_works/show.html.erb", raster_partial: true do
+  let(:resource) do
+    FactoryGirl.build(:raster_work, id: 'test',
+                                    title: ['a raster work'],
+                                    creator: ['Gerardus'],
+                                    date_created: ['1595-01-02'],
+                                    rights: ['No Warping'])
+  end
+  let(:solr_document) { SolrDocument.new(resource.to_solr) }
+  let(:ability) do
+    a = double("ability")
+    allow(a).to receive(:can?).and_return(true)
+    a
+  end
+  let(:presenter) { RasterWorkShowPresenter.new(solr_document, ability) }
+  let(:blacklight_config) { CatalogController.new.blacklight_config }
+  let(:editor) { true }
+  let(:collector) { true }
+  before do
+    stub_blacklight_views
+    assign(:presenter, presenter)
+    expect(view).to receive(:provide).with(:page_title, 'a raster work')
+    expect(view).to receive(:provide).with(:page_header).and_yield
```

Comment:
Apologies for being nitpicky about this again, but I think these should be stubbed with `allow` instead of mocked with `expect`.